recipe/meta.yaml

```
@@ -8,10 +11,11 @@ source:
 
 build:
   number: 0
-  skip: true  # [osx or win]
+  noarch: generic
 
 requirements:
   run:
+    - __{{ noarch_platforms }}
```

Looks like we may need this instead

```suggestion
    - __linux  # [linux]
    - __win    # [win]
```